torch_audiomentations/utils/object_dict.py

```diff
@@ -0,0 +1,39 @@
+# Inspired by tornado
+# https://www.tornadoweb.org/en/stable/_modules/tornado/util.html#ObjectDict
+
+try:
+    import typing
+    from typing import cast
```

Comment:
Can you explain why this is imported? 
Is this a way to detect whether one can use `typing.Dict[...]` syntax?

Reply:
Hmm, I'm not sure. I think it is a copy of tornado code from a time when it supported Python 3.4. This try except is probably not needed anymore, but it also doesn't hurt too much. Should I remove it?

Reply:
I would remove it, yes.
Note that I cherry-picked this PR in #123.

Reply:
Done